enterprise_access/apps/api_client/enterprise_catalog_client.py

```diff
@@ -67,3 +67,19 @@ def catalog_content_metadata(self, catalog_uuid, content_keys, traverse_paginati
         response = self.client.get(endpoint, params=query_params)
         response.raise_for_status()
         return response.json()
+
+    def get_content_metadata_count(self, catalog_uuid):
+        """
+        Returns the count of content metadata for a catalog.
+
+        Arguments:
+            catalog_uuid (UUID): UUID of the enterprise catalog to check.
+
+        Returns:
+            The number of content metadata for a catalog.
+        """
+
+        endpoint = self.enterprise_catalog_endpoint + str(catalog_uuid) + '/get_content_metadata/'
```

Comment:
Can we reuse code from the above method calling the same endpoint?

Reply:
agreed!

Reply:
good catch!